{{cookiecutter.project_slug}}/.osparc/bin/ooil.bash

```diff
@@ -20,7 +20,6 @@ WORKDIR="$(pwd)"
 run() {
   docker run \
     --rm \
-    --pull=always \
```

Comment:
why are you removing thisone?
Actually i tis good that always pulls the latest version! 
I had also to add it so that it would not fail

Reply:
Thanks for the explanation. 
I have re-introduced it (it's a hack I had to use to make it work with one of our old gitlab runner.